src/newtestp.py

```diff
@@ -56,103 +121,33 @@ def initUI(self):
         self._Off = tk.PhotoImage(file='Off.gif')
         self._Stat = tk.PhotoImage(file='Stat.gif')
         self._18 = tk.PhotoImage(file='18.gif')
-        self.result1 = ''
-        self.result2 = ''
-        self.result3 = ''
-        self.result4 = ''
-        self.result5 = ''
-        self.result6 = ''
-        self.result7 = ''
-        self.result8 = ''
-        self.result9 = ''
-        self.result10 = ''
-        self.result11 = ''
-        self.result12 = ''
-        self.result13 = ''
-        self.result14 = ''
-        self.result15 = ''
-        self.result16 = ''
-        self.result17 = ''
-        self.result18 = ''
-        self.result19 = ''
-        self.result20 = ''
-        self.result21 = ''
-        self.result22 = ''
-        self.result23 = ''
-        self.result24 = ''
-        self.result25 = ''
-        self.result26 = ''
-        self.result27 = ''
-        self.result28 = ''
-        self.result29 = ''
-        self.result30 = ''
-        self.result31 = ''
-        self.result32 = ''  
-        self.result33 = '' 
-        self.result34 = ''
-        self.result35 = ''
-        self.result36 = ''
-        self.result37 = ''
-        self.result38 = ''
-        self.result39 = ''
-        self.result40 = ''
-        self.result41 = ''
-
-
-        self.label1 = ''
-        self.label2 = ''
-        self.label3 = ''
-        self.label4 = ''
-        self.label5 = ''
-        self.label6 = ''
-        self.label7 = ''
-        self.label8 = ''
-        self.label9 = ''
-        self.label10 = ''
-        self.label11 = ''
-        self.label12 = ''
-        self.label13 = ''
-        self.label14 = ''
-        self.label15 = ''
-        self.label16 = ''
-        self.label17 = ''
-        self.label18 = ''  
-        self.label19 = ''
-        self.label20 = ''
-        self.label21 = ''
-        self.label22 = ''
-        self.label23 = ''   
-        self.label24 = ''
-        self.label25 = ''
+
+        self.result = {name: str() for name in RESULT_NAMES}
+        self.label = {name: str() for name in LABEL_NAMES}

```

Comment:
@Topsham Essentially, these two lines initialize the results and labels, which are now dictionaries